addon/styles/addon.scss

```diff
@@ -57,7 +57,6 @@ body > .ember-view {
   @apply docs-flex;
   @apply docs-flex-col;
 
-  @apply docs-mt-1;
```

Comment:
When you open the version selector, you could see the page jumping down a little bit (you can see that happening right now on the current docs website).

This was because opening the dropdown adds a new `.ember-view` div on the body, which gets targeted by this selector. Deleting the top margin didn't result in anything weird for me and fixed the problem.